llvm/utils/TableGen/DAGISelMatcherEmitter.cpp

```diff
@@ -1066,6 +1077,9 @@ static StringRef getOpcodeString(Matcher::KindTy Kind) {
   case Matcher::RecordMemRef: return "OPC_RecordMemRef"; break;
   case Matcher::CaptureGlueInput: return "OPC_CaptureGlueInput"; break;
   case Matcher::MoveChild: return "OPC_MoveChild"; break;
+  case Matcher::MoveSibling:
+    return "OPC_MoveSibling";
+    break;
```

Comment:
Can we delete all the unreachable `break`s in this switch as a separate patch?

Reply:
Commited as d0c8d41e103dcd3ee1b2b5ed17f9a68e973e01d6.